server.js

```diff
@@ -1,38 +1,34 @@
 // TASK 1: Setting up server
 
 
-
 const http = require('http');
 
-// Creating the HTTP server
 const server = http.createServer((req, res) => {
-
```

Comment:
The MongoDB URI should be stored as an environment variable instead of being hard-coded in the code. This is to ensure security and allow for easy configuration changes.